src/test/java/api/items/ItemApiExamples.java

```diff
@@ -403,6 +415,9 @@ public void canUpdateExistingItem()
       .forHolding(holdingId)
       .withBarcode("645398607547")
       .canCirculate()
+      .withItemLevelCallNumber(CALL_NUMBER)
+      .withItemLevelCallNumberSuffix(CALL_NUMBER_SUFFIX)
+      .withItemLevelCallNumberPrefix(CALL_NUMBER_PREFIX)
```

Comment:
If these properties are not being changed, why do they need to be included in this test?

Reply:
I just realized that this test is redundant, because PUT does not return any entity, so we just need to tests POST and GET.